recipes/libtribler/recipe.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+DEPS_libtribler=(kivy python openssl m2crypto twisted sqlite3 pyasn1 apsw cherrypy netifaces libtorrent libnacl libsodium pil plyvel requests)
+BUILD_libtribler=$BUILD_PATH/libtribler/tribler-git
+RECIPE_libtribler=$RECIPES_PATH/libtribler
+
+function prebuild_libtribler() {
+
+	# Clone repo so all submodules are included:
+	cd $BUILD_PATH/libtribler
+	git clone --recursive https://github.com/tribler/tribler.git tribler-git
```

Comment:
The problem with the archive is that it doesn't download the git submodules. I'll have a look at this and perhaps package the submodules separately, probably the preferred option, or see if I can do something about the cloning.